AmplifyPlugins/Storage/Sources/AWSS3StoragePlugin/Configuration/AWSS3PluginPrefixResolver.swift

```diff
@@ -12,6 +12,7 @@ import AWSPluginsCore
 /// Resolves the final prefix prepended to the S3 key for a given request.
 ///
 /// - Tag: AWSS3PluginPrefixResolver
+@available(*, deprecated)
```

Comment:
any meaningful messages here like the use of `StoragePath`?

Reply:
Leaving the message empty for now and until one is provided by Product team.